jobs/uaa/spec

```diff
@@ -1014,6 +1014,7 @@ properties:
         tokenUrl: <URL to the token endpoint of the provider>
         tokenKey: <Token verification key>
         tokenKeyUrl: <URL for token verification. Will be used if tokenKey is not specified.>
+        cacheJwks: <Option to enable caching for the JWKS (verification key for validating token signatures)>
```

Comment:
<Option to enable caching for the JWKS (verification key for validating token signatures)>
<Option to control caching behavior for the JWKS (verification key for validating token signatures, default true)>

Reply:
Updated based on comment.